migrations/versions/2023_04_13_661d993d5c07_add_cpu_table.py

```diff
@@ -0,0 +1,69 @@
+"""add cpu table
+
+Revision ID: 661d993d5c07
+Revises: 8e555602a405
+Create Date: 2023-04-13 21:39:53.011241
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '661d993d5c07'
+down_revision = '8e555602a405'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('cpu',
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('cores', sa.Integer(), nullable=True),
+    sa.Column('threads', sa.Integer(), nullable=True),
+    sa.Column('release', sa.Integer(), nullable=True),
+    sa.Column('multi_thread_rating', sa.Integer(), nullable=True),
+    sa.Column('single_thread_rating', sa.Integer(), nullable=True),
+    sa.PrimaryKeyConstraint('name')
+    )
+    op.execute("""
+INSERT INTO cpu(name,cores,threads,release,multi_thread_rating,single_thread_rating) VALUES
+("AMD EPYC 7401P",24,48,2017,28282,1698),
+("AMD Opteron 6338P",12,12,2014,NULL,NULL),
+("AMD Ryzen 7 1700X",8,16,2017,15691,2979),
+("AMD Ryzen 7 3700X",8,16,2019,22674,2664),
+("AMD Ryzen 7 PRO 1700X",8,16,2017,15695,2148),
+("AMD Ryzen 9 3900",12,24,2019,30663,2607),
+("AMD Ryzen Threadripper 2950X",16,32,2018,29370,2461),
+("Intel Core i5-12500",6,12,2022,19994,3722),
+("Intel Core i7-2600",4,8,2010,5329,1738),
+("Intel Core i7-3770",4,8,2012,6403,2073),
+("Intel Core i7-3930",6,12,2011,8250,1758),
+("Intel Core i7-4770",4,8,2013,7071,2171),
+("Intel Core i7-6700",4,8,2015,8084,2298),
+("Intel Core i7-7700",4,8,2016,8647,2465),
+("Intel Core i7-8700",6,12,2017,12961,2655),
+("Intel Core i9-12900K",16,24,2021,41597,4205),
+("Intel Core i9-9900K",8,16,2018,18526,2945),
+("Intel XEON E-2176G",6,12,2018,13586,2726),
+("Intel XEON E-2276G",6,12,2019,13803,2856),
+("Intel Xeon E3-1245",4,8,2011,5334,1720),
+("Intel Xeon E3-1245V2",4,8,2012,6382,2053),
+("Intel Xeon E3-1246V3",4,8,2014,7287,2212),
+("Intel Xeon E3-1270V3",4,8,2013,7322,2204),
+("Intel Xeon E3-1271V3",4,8,2014,7547,2290),
+("Intel Xeon E3-1275V6",4,8,2017,9213,2554),
+("Intel Xeon E3-1275v5",4,8,2015,8461,2354),
+("Intel Xeon E5-1650V2",6,12,2013,9333,2029),
+("Intel Xeon E5-1650V3",6,12,2014,10447,2130),
+("Intel Xeon W-2145",8,16,2017,18105,2610),
+("Intel Xeon W-2245",8,16,2020,19492,2747)
```

Comment:
This syntax fails for postgresql, as `"$name"` indicates a column or table name.

Either use `'$name'` (if supported by sqlite) or switch to a programatical approach :)

Reply:
I renamed the property to cpu_name to be safe

Reply:
Ok, a bit of miscommunication.

This is actually about the quotes. Double quotes `"` have a special meaning in postgres, indicating column or table names. Strings always start with a single quote `'`

Reply:
done